.github/workflows/generate_translations.yml

```diff
@@ -44,15 +44,14 @@ jobs:
           ref: ${{ github.head_ref }}
           token: ${{ secrets.ADG_THEME_ACCESS_TOKEN }}
 
-      - name: Add comment if cloning fails
-        uses: mshick/add-pr-comment@v1
+      - name: Checkout master as feature branch does not exist
+        uses: actions/checkout@v2
+        continue-on-error: true
         with:
-          message: |
-            Cloning failed. :x:
-            No branch ref `${{ github.head_ref }}` found on adg-edx-theme repo.
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          repo-token-user-login: 'github-actions[bot]'
-          allow-repeats: true
+          repository: OmnipreneurshipAcademy/adg-edx-theme
+          path: ./themes
+          ref: master
```

Comment:
Lets use `${{ github.base_ref }}` in place of master.

Reply:
This step will run if there is no feature branch in theme. Hence we specifically need to use master as reference. Also this action won't run in PRs against develop hence we don't need to use base branch.